### ISSUE TITLE
Add Google Analytics tracking to the layout.

### ISSUE THREAD

**Issue by ryancwalsh** [NEAR-Edu/near-certification-tools#15]
You'll probably need to work with Sherif (get him to give you whatever tracking ID you need).

Hiba knows about G.A. and has offered to help if you need.

**Comment by amgando** [comment]
why are we adding GA to this tool?  in my mind this is something internal we're using to create certifications.  certs will also have a shareable link for social sharing.  is that the idea behind measuring use?  makes sense but seems low priority to me at this point

**Comment by ryancwalsh** [comment]
@amgando Yeah the GA would be not for the issuing side but for the gallery / viewing side. And yes it's low priority.

**Comment by ozanisgor** [comment]
@ryancwalsh I added GA with https://github.com/NEAR-Edu/near-certification-tools/pull/43. 
Now the only missing thing is `tracking ID` that I need to ask from Sherif right?

**Comment by ryancwalsh** [comment]
@ozanisgor Did you ever get the tracking ID? 

If yes, could you please send it to me via Discord DM (since it's not super secret)? Thanks.

If not, no worries, I'll get it.

**Comment by ozanisgor** [comment]
@ryancwalsh I could not get it before I leave because Shashi was sick that days but I informed Shashi and he said he will sync with you and share whatever is required.

**Comment by ryancwalsh** [comment]
@ozanisgor Thanks!
